### PR TITLE
Add I18n support

### PR DESCRIPTION
**Pull request by santib**
- [x] explain solution in README
- [x] add missing translations
- [x] add script that runs `bundle exec ruby i18n_pages.rb && bundle exec jekyll s`

maybe later..
- [x] add github action that runs `bundle exec ruby i18n_pages.rb --all` and fails if there are differences

### CODE REVIEW

**Review by francobottero — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,3 +15,21 @@ make server
...
+We want to support Spanish as the default language, and English as an alternative so that we can reach most places in the world.
+
+The solution includes:
+- Language switcher in the UI
+  - By default Spanish
+  - URLs prefixed with /en/ for English
+- Translating text in pages
+-   Add translated text in YML files under `_data/i18n/[lang].yml`
+-   Calling the translated texts with `site.data.i18n[page.locale].whatever_key_in_your_yml_file`
+-   Note: `page.locale` is automatically set in every page
```
Comment (santib) [inline README.md:29]:
```suggestion
  - Add translated text in YML files under `_data/i18n/[lang].yml`
  - Calling the translated texts with `site.data.i18n[page.locale].whatever_key_in_your_yml_file`
  - Note: `page.locale` is automatically set in every page
```

File: README.md [hunk 2]
```diff
@@ -15,3 +15,21 @@ make server
...
+- Translating text in pages
+  - Add translated text in YML files under `_data/i18n/[lang].yml`
+  - Calling the translated texts with `site.data.i18n[page.locale].whatever_key_in_your_yml_file`
+  - Note: `page.locale` is automatically set in every page
+- A script to automatically generate translated files
+  - The script is ran in the pre-commit hook so it's transparent for devs
+  - You can still run `bundle exec ruby i18n_pages.rb --all` to re-generate all the translated files
+  - It automatically translate links so to navigate the site in the same language
+  - It automatically translate collections
+  - A binary `rubyuy-server` to automatically run the script before starting the Jekyll server
```
Comment (santib) [inline README.md:35]:
```suggestion
- A binary `rubyuy-server` to automatically run the script before starting the Jekyll server
```

File: README.md [hunk 3]
```diff
@@ -15,3 +15,21 @@ make server
...
    1. If you have them in Google Slides, you can go to File > Share > Publish to web > Embed > Publish and get the url of the src of the iframe
+
+## i18n
+We want to support Spanish as the default language, and English as an alternative so that we can reach most places in the world.
+
+The solution includes:
+- Language switcher in the UI
+  - By default Spanish
+  - URLs prefixed with /en/ for English
+- Translating text in pages
```
Comment (mwolman) [inline README.md:26]:
```suggestion
- Translating text on pages
```

File: README.md [hunk 4]
```diff
@@ -15,3 +15,21 @@ make server
...
+  - Add translated text in YML files under `_data/i18n/[lang].yml`
+  - Calling the translated texts with `site.data.i18n[page.locale].whatever_key_in_your_yml_file`
+  - Note: `page.locale` is automatically set in every page
```
Comment (mwolman) [inline README.md:29]:
```suggestion
  - Note: `page.locale` is automatically set on every page
```

File: README.md [hunk 5]
```diff
@@ -15,3 +15,21 @@ make server
...
+- A script to automatically generate translated files
+  - The script is ran in the pre-commit hook so it's transparent for devs
```
Comment (mwolman) [inline README.md:31]:
```suggestion
  - The script is run in the pre-commit hook, so it's transparent for devs
```

File: README.md [hunk 6]
```diff
@@ -15,3 +15,21 @@ make server
...
+  - You can still run `bundle exec ruby i18n_pages.rb --all` to re-generate all the translated files
+  - It automatically translate links so to navigate the site in the same language
```
Comment (mwolman) [inline README.md:33]:
```suggestion
  - It automatically translates links to navigate the site in the same language
```

File: README.md [hunk 7]
```diff
@@ -15,3 +15,21 @@ make server
...
+  - It automatically translate collections
```
Comment (mwolman) [inline README.md:34]:
```suggestion
  - It automatically translates collections
```

**Review by mwolman — COMMENTED** [review]
I leave some Grammarly suggestions 😃

**Review by juanmferreira93 — APPROVED** [review]

**Review by mwolman — APPROVED** [review]

**Pull request closed without merging**